ci/citest/acceptance-test/multibox/ind-steps/step-qemu-kvm/install.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 
+# install_yum_package "qemu-system-x86"
```

Comment:
Why is this commented out? Perhaps because it gets installed as a dependency? Personally I'm not a fan of including commented code since old code is preserved in the git history anyway but not gonna hold back on merging for that.